src/todos/todos.controller.ts

```diff
@@ -1,4 +1,4 @@
-import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Param, Post } from '@nestjs/common';
```

Comment:
Let's fix the imports:

Library imports at the top

Then @ imports

Then relative imports